src/App/Header/UserMenu/Menu.tsx

```diff
@@ -18,20 +18,16 @@ export default function Menu({
       className={`${classes} bg-white dark:bg-blue-d6 w-max rounded overflow-hidden`}
     >
       <p className="text-sm p-2 text-gray-d1 dark:text-gray">
-        Welcome, {user?.attributes?.given_name}!
+        Welcome, {user?.attributes?.given_name || user?.attributes?.email}!
```

Comment:
If `user.attributes` is null/undefined, this would still result in `Welcome, !`

Reply:
Whoops! Good catch, forgot to update the rest.

Reply:
Fixed in a21a816e7